src/Basics/constantsAndFunctions.jl

```diff
@@ -7,19 +7,21 @@
 
 
 # Epsilon and sign
-const neps = sqrt( eps(Modia3D.MPRFloatType) )
+const neps = sqrt( eps() )
 
-function sign_eps(value::T; ) where {T}
-    seps::T = 100.0*neps
+nepsMPR(::Type{T}) where {T} = sqrt( eps(T) )
```

Comment:
Function name should not include "MPR" because function is also used non MPR code.

Reply:
I changed that. In MPR code, including bounding boxes only `nepsMPR()` is used. Everything else uses `Modia3D.neps`

Reply:
But isn't bounding box computation independent of MPR?

Reply:
There is the following issue: both, bounding box computation and MPR computation use support point calculation. Therefore, both call e.g. `supportPoint_abs_Cylinder`. Maybe we should discuss this offline?

Reply:
That is the point: When support point calculation is called from bounding box computation, nepsMPR is misleading. You could rename it to nepsType or so.

Reply:
Ok, I'll change that.